### DISCUSSION TITLE
Send canned message from bot

### DISCUSSION THREAD

**Discussion by alonsoac** [LiveHelperChat/livehelperchat#2104] [General]
Hi. Is it possible to send canned message from bot?

**Comment by remdex** [comment]
why would you need that if you can send anything you want?

**Reply by alonsoac** [reply]
The texts and attachments that I need to send are already in canned messages. Agents have access to edit canned messages and they can change them at any moment. So if the bot could send that, everything is great.
If not, then I would need to give them access to edit either the bot or the Bot individualization and they would have to edit the text in 2 different places each time.

**Reply by remdex** [reply]
Hi,
Thanks for reasoning it. I'll think about how to add option to send a canned message from bot.
